packages/shared/src/utilities/color/isRgbaLongFormat.ts

```diff
@@ -0,0 +1,6 @@
+import { Color } from '@frontify/fondue';
```

Comment:
```suggestion
/* (c) Copyright Frontify Ltd., all rights reserved. */

import { Color } from '@frontify/fondue';
```

Reply:
👍

Reply:
removed the file again, as I actually don't need the isRgbaLongFormat.ts in this case